tests/validators/test_list.py

```diff
@@ -397,6 +397,23 @@ def f(v: int) -> int:
     )
 
 
+def test_list_fail_fast():
+    s = core_schema.list_schema(core_schema.int_schema(), fail_fast=True)
+    v = SchemaValidator(s)
+
+    with pytest.raises(ValidationError) as exc_info:
+        v.validate_python([1, 'not-num', 'again'])
+
+    assert exc_info.value.errors(include_url=False) == [
+        {
+            'type': 'int_parsing',
+            'loc': (1,),
+            'msg': 'Input should be a valid integer, unable to parse string as an integer',
+            'input': 'not-num',
+        }
+    ]
+
```

Comment:
It might be nice to have a second assertion here which confirms that if `fail_fast=False` then both errors are returned.

Reply:
Thanks, fixed